libdnf/rpm/package_sack.cpp

```diff
@@ -208,7 +208,7 @@ void PackageSack::Impl::write_ext(
     auto libsolv_repo = libsolv_repo_ext.repo;
     const char * repo_id = libsolv_repo->name;
 
-    assert(repodata_id);
+    libdnf_assert(repodata_id > 0, "TODO");
```

Comment:
Just describe what happened, and where it happened.
Please do not change conditions. `assert(repodata_id)` is an alternative to assert(repodata_id != 0), but it is not assert(repodata_id > 0). `repodata_id` variable has type Id == `Int` and `int` can have also negative values.

Reply:
What about:
`libdnf_assert(repodata_id != 0, "TODO");`